misc/formatter/modules/formatter-core/src/test/resources/expressions/raw-template/source/regex_expressions.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/constraint;
```

Comment:
We don't use standard library modules in lang tests.

1. introduces a cyclic dependency between lang and the stdlib module
2. can result in build failures
 i. due to bug fixes that cause the stdlib module builds to fail - need to wait for them to be fixed
 ii. local build failures due to incompatible BIR caches

Reply:
please check now.